pkgs/build-support/trivial-builders/default.nix

```diff
@@ -380,6 +383,8 @@ rec {
           runHook postCheck
         ''
         else checkPhase;
+    } // optionalAttrs (postBuild == null) {
```

Comment:
```suggestion
    } // optionalAttrs (postBuild != null) {
```

Reply:
tests.writeTextFile on x86_64-linux Skipped — [No attempt](https://github.com/NixOS/nixpkgs/pull/243023/checks?check_run_id=18100314083)

Apparently this test is not getting run as part of CI?

Reply:
> tests.writeTextFile on x86_64-linux Skipped — No attempt

It should be `pkgs.tests.trivial-builders.writeTextFile`. I made it wrong when calling OfBorg to build.